olive/model/handler/mixin/hf_config.py

```diff
@@ -45,7 +44,7 @@ def get_hf_model_config(self):
         if self.hf_config is None:
             raise ValueError("HF model_config is not available")
 
-        return get_hf_model_config(self._get_model_path_or_name(), **HfConfigMixin._get_loading_args(self.hf_config))
+        return get_hf_model_config(self._get_model_path_or_name(), **get_loading_args(self.hf_config))
```

Comment:
The static methods are not needed and after talking with @guotuofeng, I already have dev branch that removed these changes https://github.com/microsoft/Olive/commit/b4c2700f0b3a3ca84768f1196a08b468c827a47a 

Just working on updating the unit tests to take care of this.

Reply:
PR with the above changes https://github.com/microsoft/Olive/pull/807

Reply:
I haven't removed them. Just moved them to utils. Could you push on top of this PR I.e. once this gets merged.

Reply:
Code rebased to include changes in PR #807